Code/GraphMol/ChemTransforms/ChemTransforms.h

```diff
@@ -265,6 +265,15 @@ RDKIT_CHEMTRANSFORMS_EXPORT void parseQueryDefText(
     std::map<std::string, ROMOL_SPTR> &queryDefs, bool standardize = true,
     const std::string &delimiter = "\t", const std::string &comment = "//",
     unsigned int nameColumn = 0, unsigned int smartsColumn = 1);
-}  // namespace RDKit
+//! \brief Updates conformer coordinates in res from those in mol
```

Comment:
I think this needs some more explanation, i.e. the atoms must be in the same order (except for the removed atoms) and the output is undefined is res wasn't created from mol.  I don't think this function is suitable for the public API (I can change my mind of course)

I would rather have an updatesubmol(mol, res, INT_MAP_INT) function ala PathToSubmol with maybe a helper, updatesubmol(mol, res, pathFromDeletedAtoms(mol, res, deleted_atoms))

Reply:
I'm inclined not to make `updateSubConfs` public, unless there are other scenarios where it might be useful.

Reply:
I'm definitely in favor of putting it in a `details` namespace.